specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json

```diff
@@ -0,0 +1,2651 @@
+            "$ref": "#/definitions/OperationEntity"
+          }
+        }
+      }
+    },
+    "OperationEntity": {
+      "description": "The operations supported by Bot Service Management.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}.",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplayInfo",
+          "description": "The operation supported by Bot Service Management."
+        },
+        "origin": {
+          "description": "The origin of the operation.",
+          "type": "string"
+        },
+        "properties": {
+          "description": "Additional properties.",
+          "type": "object"
+        }
+      }
+    },
+    "OperationDisplayInfo": {
+      "description": "The operation supported by Bot Service Management.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The description of the operation.",
+          "type": "string"
+        },
+        "operation": {
+          "description": "The action that users can perform, based on their permission level.",
+          "type": "string"
+        },
+        "provider": {
+          "description": "Service provider: Microsoft Bot Service.",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Resource on which the operation is performed.",
+          "type": "string"
+        }
+      }
+    },
+    "CheckNameAvailabilityRequestBody": {
+      "description": "The request body for a request to Bot Service Management to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "the name of the bot for which availability needs to be checked.",
+          "type": "string"
+        },
+        "type": {
+          "description": "the type of the bot for which availability needs to be checked",
+          "type": "string"
+        }
+      }
+    },
+    "CheckNameAvailabilityResponseBody": {
+      "description": "The response body returned for a request to Bot Service Management to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "valid": {
+          "description": "indicates if the bot name is valid.",
+          "type": "boolean"
+        },
+        "message": {
+          "description": "additional message from the bot management api showing why a bot name is not available",
+          "type": "string"
+        }
+      }
+    },
+    "WebAppBotSettings": {
+      "description": "The response body returned for a request to Bot Service Management to check per subscription WebAppBotSettings",
+      "type": "object",
+      "properties": {
+        "AzureWebJobsBotFrameworkDirectLineEndpoint": {
+          "description": "web app setting for Bot Framework SDK",
+          "type": "string"
+        },
```

Comment:
Is this one necessary? It's not used in the SDK, so I think this might be only be applicable to the Service team's environments

Reply:
removed